content/sensu-go/5.17/guides/scale-event-storage.md

```diff
@@ -178,8 +179,86 @@ You may see old events in the web UI or sensuctl output until the etcd datastore
 
 _**NOTE**: If your Sensu Go license expires, event storage will automatically revert to etcd._
 
+## Configure Postgres Streaming Replication
```

Comment:
Note this is for EL7. The commands might change a bit depending on platform.

Reply:
Added a note that the paths and service names are OS dependent and may vary based on operating systems